skorch/callbacks/logging.py

```diff
@@ -62,7 +89,8 @@ class PrintLog(Callback):
     ----------
     keys_ignored : str or list of str (default=None)
       Key or list of keys that should not be part of the printed
-      table. Note that keys ending on '_best' are also ignored.
+      table. Note that keys starting with 'even_' or ending on '_best'
```

Comment:
```suggestion
      table. Note that keys starting with 'event_' or ending on '_best'
```

Note that this description is incomplete. The new filtering function also ignores `_batch_count` keys (line 41), and `PrintLog` adds `batches` key to the ignored list  (line 136).